Update getting-started.md (#2028)
saucelabs/sauce-docs: docs/orchestrate/getting-started.md

> This page outlines how to run your browser and mobile tests in Sauce Orchestrate.

Reviewer: ```suggestion
This page outlines how to run your browser and mobile tests in Sauce Orchestrate.

:::note
Sauce Orchestrate is available as an add-on for Enterprise plans only. If you want to upgrade your Enterprise plan to use Sauce Orchestrate, contact your Customer Success Manager or Support at help@saucelabs.com.
:::
```

> This page outlines how to run your browser and mobile tests in Sauce Orchestrate.

Reviewer: If SO is for enterprise plans only, we should also add a badge at the top of the page to clarify.
```suggestion
<p><span className="sauceGreen">Enterprise Plans only</span></p>

This page outlines how to run your browser and mobile tests in Sauce Orchestrate.
```

Author: @lysannep this isn't ONLY for enterprise, it's a paid add-on all customers can access

## What You'll Need

> :::caution Please ensure your organization is enrolled to use Sauce Orchestrate otherwise you will encounter an error. If you are not enrolled, please feel free to reach out to your Customer Success Manager or Support at help@saucelabs.com :::

Reviewer: ```suggestion

:::note 
Make sure your organization is enrolled to use Sauce Orchestrate, otherwise you will encounter an error. If you are not enrolled, contact your Customer Success Manager or Support at help@saucelabs.com. 
:::
```

> :::caution Please ensure your organization is enrolled to use Sauce Orchestrate otherwise you will encounter an error. If you are not enrolled, please feel free to reach out to your Customer Success Manager or Support at help@saucelabs.com :::
> 

Reviewer: We could move the note higher up the page.
```suggestion
```

Author: wherever is best

## What You'll Need

> :::note 

Reviewer: ```suggestion
:::note
```
